src/structures/Role.js

```diff
@@ -54,7 +54,7 @@ class Role {
     this.position = data.position;
 
     /**
-     * The evaluated permissions number
+     * The permissions number, taking into account roles and permission overwrites.
```

Comment:
This is on the Role object, which does not take into account roles and permission overwrites. Just the role's own permissions.